rasterio/rio/merge.py

```diff
@@ -13,6 +12,7 @@
 from . import options
 import rasterio
 from rasterio.transform import Affine
+from rasterio.tools.merge import merge as merge_tool
```

Comment:
Numpy takes too much time to import and this is why I was importing it inside the merge command, to not burden commands that didn't need numpy. Let's similarly import `rasterio.tools.merge` inside the command.